code/game/objects/structures/barsigns.dm

```diff
@@ -1,4 +1,4 @@
-/obj/structure/sign/barsign // All Signs are 64 by 32 pixels, they take two tiles
+/obj/machinery/barsign // All Signs are 64 by 32 pixels, they take two tiles
```

Comment:
should probably move this out of the structures folder

Reply:
Good idea!  Applied with https://github.com/tgstation/tgstation/pull/73106/commits/31d48de1f25a15cf9dae6a865a47685a25b413d3